core/src/epicli/data/common/defaults/configuration/shared-config.yml

```diff
@@ -7,5 +7,8 @@ specification:
   download_directory: /tmp # directory where files and images will be stored just before installing/loading
   vault_location: '' # if empty "BUILD DIRECTORY/vault" will be used
   vault_tmp_file_location: SET_BY_AUTOMATION
+  supported_architecture:
```

Comment:
What's about x86_64?

Reply:
not needed for now.

Reply:
it is done as assumed in the task:
aarch64: [] -> this empty list means ARM is supported by none of components  so it will fail everytime we apply epi on the machine with ARM arch. Once we support minimal list of components - it should be added here as starting point.

missing x86_64 -> not figuring arch means this arch is supported by all components so no point to list them.